jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/FilesTask.java

```diff
@@ -149,9 +150,8 @@ public void listFiles() {
     printProjectFiles(project);
 
     // Print extra layer
-    if (Files.exists(jibExtension.getExtraDirectory().getPath())) {
-      System.out.println(jibExtension.getExtraDirectory().getPath());
-    }
+    List<Path> extraDirectories = jibExtension.getExtraDirectory().getPaths();
```

Comment:
Do you think it's worth renaming `getExtraDirectory()` to `getExtras()`?

Reply:
I saw your [comment](https://github.com/GoogleContainerTools/jib/issues/1020#issuecomment-437902030) before, and I thought `jib.extras` sounded too general and also `getExtras()` in our code could be confusing. And `getExtraDirectory()` inside `JibExtension` seems consistent with that it returns the `ExtraDirectoryParameters` object, which matches the config parameter name.

Reply:
We may rename this later, but to reduce complexity and potential mistakes, I suggest stop changing names in this PR as it is in working order now.